setup.py

```diff
@@ -1,13 +1,28 @@
-from setuptools import setup
-
-setup(
-    name = 'aiocfscrape',
-    packages = ['aiocfscrape'],
-    version = '0.0.9',
-    description = 'A simple async Python module to bypass Cloudflare\'s anti-bot page. See https://github.com/pavlodvornikov/aiocfscrape for more information.',
-    author = 'pavlodvornikov',
-    author_email = 'pavlodvornikov@gmail.com',
-    url = 'https://github.com/pavlodvornikov/aiocfscrape',
-    keywords = ['cloudflare', 'scraping', 'asyncio'],
-    install_requires = ['js2py==0.59', 'aiohttp >= 3.1.3']
-)
+from setuptools import find_packages, setup
+
+version = '0.0.10.dev0'
+
+
+long_description = '{}\n\n{}'.format(open('README.rst').read(),
+                                     open('CHANGES.rst').read())
+
+setup(name='aiocfscrape',
+      version=version,
+      description='A simple async Python module to bypass Cloudflare\'s anti-bot page. '
+                  'See https://github.com/pavlodvornikov/aiocfscrape for more information.',
+      long_description=long_description,
+
+      author='pavlodvornikov',
+      maintainer='Nachtalb',
+      url='https://github.com/pavlodvornikov/aiocfscrape',
+      license='MIT',
+
+      packages=find_packages(exclude=['ez_setup']),
+      include_package_data=True,
+      zip_safe=False,
+
+      install_requires=[
+          'js2py',
+          'aiohttp >= 3.1.3, <4a',
```

Comment:
According to docs ([1](https://docs.python.org/3/library/typing.html?#typing.final), [2](https://www.python.org/dev/peps/pep-0591/)) 'final' decorator does not affect runtime checking. I believe this means that ClientSession can still be inherited and used. Only static code analysis tools like pylint or the one integrated to IDE will warn that ClientSession "should not be inherited".

Reply:
> Since version 4 of aiohttp it is forbidden to inherit from ClientSession
> aio-libs/aiohttp#3933

https://github.com/aio-libs/aiohttp/blob/15e5cbb05766bdf647c04a161d940614c434dbb1/aiohttp/client.py#L260-L262

Reply:
Oh I see, didn't notice this one